gdx/src/com/badlogic/gdx/graphics/glutils/IndexArray.java

```diff
@@ -28,10 +28,19 @@
 	ShortBuffer buffer;
 	ByteBuffer byteBuffer;
 
+	// used to work around bug: https://android-review.googlesource.com/#/c/73175/
+	private final boolean empty;

```

Comment:
Mhmmm... actually, "maxIndices" is not a field either, but would have to be to be checked in the getters. So it's either "maxIndices" or "empty" which would need to be added as a field. Actually, I tend to find my original code more self-explanatory :-] Please tell me how to proceed.